doc/code/templates.rst

```diff
@@ -64,6 +61,8 @@ Each trainable template has a dedicated function in :mod:`pennylane.init` which
 The following parameter initialization methods are available:
 
 
+.. rst-class:: contents local topic
+
 .. toctree::
     :maxdepth: 3
```

Comment:
The above change is a slight bugfix - by removing the "Creating initial parameters" section title, we allow the TOC here to show up in the sidebar (a quirk of sphinx)